### PR TITLE
Maint-27536: Fix space members Invite button translation issue

### PR DESCRIPTION
**Pull request by Jihed525**
*Add key in order to translate button label
*Change position so if invite button label is too long, the user mention input field wouldn't be displayed under the invite button

### CODE REVIEW

File: extension/war/src/main/webapp/groovy/social/webui/UIUserInvitation.gtmpl [hunk 1]
```diff
@@ -27,10 +28,10 @@ placeholder = placeholder?.replace("'", "\\'").replace("&#39;", "\\&#39;");
 
   <% uiform.begin();%>
   <div class="uiGrayLightBox">
-		<div class="pull-right"><button class="btn" type="button" onclick="$inviteAction">Invite</button></div>
-		<div class="mention-input">
-			<% uiform.renderField("user"); %>
-		</div>
+        <div class="mention-input">
```
Comment (azayati) [inline extension/war/src/main/webapp/groovy/social/webui/UIUserInvitation.gtmpl]: @Jihed525 can you please rectify indentation here ?

File: extension/war/src/main/webapp/groovy/social/webui/UIUserInvitation.gtmpl [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <%
 String inviteAction = uicomponent.event("Invite") + "; return false;";
+String inviteSpaceMember = _ctx.appRes("UISpaceMember.button.invite");
```
Comment (azayati) [inline extension/war/src/main/webapp/groovy/social/webui/UIUserInvitation.gtmpl:3]: Do we already have the value of this key ?

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: extension/war/src/main/webapp/groovy/social/webui/UIUserInvitation.gtmpl (hunk 1 above)
Comment (Jihed525) [reply extension/war/src/main/webapp/groovy/social/webui/UIUserInvitation.gtmpl]: ok

File: extension/war/src/main/webapp/groovy/social/webui/UIUserInvitation.gtmpl (hunk 2 above)
Comment (Jihed525) [reply extension/war/src/main/webapp/groovy/social/webui/UIUserInvitation.gtmpl:3]: Absolutely ([link](https://github.com/Meeds-io/social/blob/develop/component/webui/src/main/resources/locale/social/Webui_en.properties#L757))

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**